docs/fix/forbid-unreachable.rst

```diff
@@ -0,0 +1,15 @@
+fix/forbid-unreachable
```

Comment:
I suggest actually calling this script just `forbid` and make it a general purpose productivity tool instead of a "bugfix". It could have the following syntax:

`forbid` - print out the number of forbidden items per type (and a total at the end)
`forbid all` - forbids all items (useful if player wants to go selectively unforbid just a few items)
`forbid unreachable` - forbids unreachable items

with tags: `fort`, `productivity`, `items`

Reply:
That was fun!